src/useq/_plate.py

```diff
@@ -63,12 +63,30 @@ def _to_slice(value: Any) -> slice:
 
 
 class WellPlate(FrozenModel):
-    """A multi-well plate definition."""
+    """A multi-well plate definition.
+
+    Parameters
+    ----------
+    rows : int
+        The number of rows in the plate.
+    columns : int
+        The number of columns in the plate.
+    well_spacing : tuple[float, float] | float
+        The spacing between wells in the x and y directions.
+        If a single value is provided, it is used for both x and y.
+    well_size : tuple[float, float] | float
+        The size of each well in the x and y directions.
+        If a single value is provided, it is used for both x and y.
+    circular_wells : bool
+        Whether wells are circular (True) or rectangular (False).
+    name : str
+        A name for the plate.
+    """
 
     rows: int
     columns: int
     well_spacing: Tuple[float, float]  # (x, y)
-    well_size: Union[Tuple[float, float], None] = None  # (x, y)
+    well_size: Tuple[float, float]  # (width, height)
```

Comment:
my reasoning for making this optional was that it's only necessary if you want to take a few images per well within some area.  It's obviously pretty important in that case... but if you just wanted to take one image per well it's not necessary.  I do guess in that case someone could just put in a fake number if they don't care about where the edge of the well is?  (I do see that it makes life harder elsewhere if you can't rely on it)

fine with me to make it required though